webapp/channels/src/components/admin_console/message_export_settings.tsx

```diff
@@ -246,7 +269,7 @@ export default class MessageExportSettings extends AdminSettings {
                             id='admin.service.complianceExportDesc'
                             defaultMessage='When true, Mattermost will export all messages that were posted in the last 24 hours. The export task is scheduled to run once per day. See <link>the documentation</link> to learn more.'
                             values={{
-                                link: (msg) => (
+                                link: (msg: React.ReactNode) => (
```

Comment:
nit : can you import this from react `msg: ReactNode`

Reply:
sure done: https://github.com/mattermost/mattermost/pull/24933/commits/5123cf94c50d4b635b6d814c9c2f0e7099028e68